python/setup.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+import pathlib
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+
+
+extension = Extension(
+    name="pyoorb",
+    sources=["pyoorb.f90", "pyoorb.pyf"],
```

Comment:
NB these don't have any effect at all, since we just call `Make`. But setting non-empty `sources` is required by the Extension constructor, so I do it anyway.